rts/motoko-rts/src/lib.rs

```diff
@@ -26,12 +26,18 @@ pub mod continuation_table;
 #[cfg(feature = "ic")]
 mod float;
 pub mod gc;
+//#[cfg(feature = "ic")]
+mod ic0_stable;
 #[cfg(feature = "ic")]
 mod idl;
 pub mod leb128;
 mod mem_utils;
 pub mod memory;
 pub mod principal_id;
+//#[cfg(feature = "ic")]
```

Comment:
This is related to the question on whether having RTS unit testing for the regions (comment below).